lib/msal-core/src/AuthResponse.ts

```diff
@@ -12,6 +13,7 @@ export type AuthResponse = {
     tenantId: string;
     tokenType: string;
     idToken: IdToken;
```

Comment:
We will be changing idToken to a string in a future release correct?

Reply:
Yes, in a "major" release as this will be a breaking change.